pom.xml

```diff
@@ -1,41 +1,60 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.1.1</version>
-		<relativePath/> <!-- lookup parent from repository -->
-	</parent>
-	<groupId>jewellery.inventory</groupId>
-	<artifactId>jewellery-inventory</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
-	<name>jewellery-inventory</name>
-	<description>Project to help jewellery businesses digitalize their inventory management</description>
-	<properties>
-		<java.version>17</java.version>
-	</properties>
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-web</artifactId>
-		</dependency>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.1.1</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>jewellery.inventory</groupId>
+    <artifactId>jewellery-inventory</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>jewellery-inventory</name>
+    <description>Project to help jewellery businesses digitalize their inventory management</description>
+    <properties>
+        <java.version>17</java.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
```

Comment:
Let's make sure we are not changing formatting of previous code. We might want to introduce [google-java-formatting](https://stackoverflow.com/a/42979701/13062206) for intellij so that when each one of us formats the code, it doesn't end up changing something we haven't touched but let's do that as a separate PR.